documentation/packages/reverse-proxy-traefik/README.md

```diff
@@ -0,0 +1,151 @@
+---
+description: Reverse proxy for secure traefik configurations.
+---
+
+# Reverse Proxy Traefik
+
+&#x20;
+
+# Reverse Proxy Traefik
+
+The package is an alternative reverse proxy nginx, this reverse proxy exposes packages using both subdomains and subdirectory to host the following services:
+
+| Package  | Hosted                                                                                                 |
+| -------- | ------------------------------------------------------------------------------------------------------ |
+| Superset | Sub Domain (e.g. superset.<domain>)                                                                    |
+| Jempi    | Sub Domain (e.g. jembi.<domain>)                                                                       |
+| Santempi | Sub Domain (e.g. santempi.<domain>)                                                                    |
+| Kibana   | Sub Domain (e.g. kibana.<domain>)                                                                      |
+| Minio    | Sub Directory (e.g. <domain>/minio)                                                                    |
+| Grafana  | Sub Directory (e.g. <domain>/grafana)                                                                  |
+| JSReport | Sub Directory (e.g. <domain>/jsreport)                                                                 |
+| OpenHim  | Sub Domain (Frontend) Sub Directory (Backend) (e.g. openhim.<domain> and openhim.<domain>/openhimcore) |
+
+> Please ensure that the ENV "DOMAIN_NAME_HOST_TRAEFIK" is set, in this documentation we will be using the place holder "domain" for its value
+
+## Domain Based Reverse Proxy
+
+The following packages do not support subdomain and require the uses of domain/subdomain to access over the reverse proxy
+
+### Superset
+
+Set the following environment variable in the package-metadata.json in the "./dashboard-visualiser-superset" directory
+
+```bash
+"environmentVariables":
+{
+# Other Configurations
+...
+    "SUPERSET_TRAEFIK_SUBDOMAIN": "superset"
+}
+```
+
+### Jempi
+
+Set the following environment variables in the package-metadata.json in the "./client-registry-jempi" directory
+
+```bash
+"environmentVariables":
+{
+# Other Configurations
+...
+    "REACT_APP_JEMPI_BASE_API_HOST": "jempi-api.domain",
+    "REACT_APP_JEMPI_BASE_API_PORT": "443",
+    "JEMPI_API_TRAEFIK_SUBDOMAIN": "jempi-api",
+    "JEMPI_WEB_TRAEFIK_HOST_NAME": "jempi-web",
```

Comment:
Shouldn't the `REACT_APP_JEMPI_BASE_API_HOST` variable be the jempi subdomain and then have the API as a path/subdirectory under the subdomain?

Reply:
Before the variable REACT_APP_JEMPI_BASE_API_HOST was just the domain name of the api e.g. jempi.org with no other path
since we are using subdomains it will use the prefix (jempi-api) and the domain (.domain)